<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionItemWidget</class>
 <widget class="QWidget" name="VersionItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Open Sans;
font-weight: 500;
font-size: 20px;
line-height: 28px;
letter-spacing: 0%;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="ItemHeader" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../../Resources/Resources.qrc">:/images/Record/version-history-icon.svg</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="versionLabel">
        <property name="styleSheet">
         <string notr="true">color: #1A1A1A;</string>
        </property>
        <property name="text">
         <string>2.0.8.9.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="linkButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../Resources/Resources.qrc">
          <normaloff>:/images/Project/expend-window.svg</normaloff>:/images/Project/expend-window.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>404</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="dateLabel">
        <property name="styleSheet">
         <string notr="true">color: #4D4D4D;
font-size: 16px;</string>
        </property>
        <property name="text">
         <string>23/09/2025</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descLabel">
     <property name="styleSheet">
      <string notr="true">color: #4D4D4D;
font-size: 16px;</string>
     </property>
     <property name="text">
      <string>Lorem ipsum dolor sit amet consectetur. Mauris mauris in sit feugiat mus massa a molestie. Id sit nisi arcu dignissim pretium. Nulla pellentesque sed leo pellentesque sed leo orci See more.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
